README.md

```diff
@@ -6,6 +6,9 @@ A module for converting cryptographic keys into human-readable phrases, similar
 to [Diceware](http://world.std.com/~reinhold/diceware.html) but with a larger
 wordlist. With Niceware, a 128-bit key is equivalent to an 8-word phrase.
 
+The wordlist contains 65,536 English words, which results in
```

Comment:
Thanks! I think this first part is redundant given the change in 57568c024adca02a6c73629f1530f2b059ebf1db.